docker/ol-docker-start.sh

```diff
@@ -28,8 +28,10 @@ su postgres -c "/etc/init.d/postgresql start"
 su openlibrary -c "scripts/infobase-server conf/infobase.yml 7000" &
 
 # wait unit postgres is ready, then reindex solr
+# also block until reindexing is completed since later services require psql
```

Comment:
https://github.com/internetarchive/openlibrary/pull/2990/files#diff-55adfd70b03f47a453f7877482fbadadR31-R48

Some of these supporting docker changes may be useful as a separate PR (for future reference)

Reply:
Going to make a new PR with cherry picked commits from this branch. I don't think things will get too wonky if I do that and revert these commits on this branch.